### PR TITLE
fix: Include path to typings dir in src/index

### LINKED ISSUE #321: Types are erronous in 5.0.1
**Description**

Proper continuation of the conversation in #312

### PR DESCRIPTION
**Pull request by rkrasiuk**
Aimed to fix #321 

Reference in index.ts includes typings/*.d.ts files to the build

FYI: Encountered the following error while trying to build project with `gitlab` as a dependency.
```bash
....
node_modules/gitlab/dist/index.d.ts:26:38 - error TS2304: Cannot find name 'Bundle'.

26 export declare const ProjectsBundle: Bundle<{
                                        ~~~~~~

node_modules/gitlab/dist/index.d.ts:69:30 - error TS2304: Cannot find name 'Bundle'.

69 export declare const Gitlab: Bundle<typeof APIServices, "Groups" | "GroupAccessRequests" | "GroupBadges" | "GroupCustomAttributes" | "GroupIssueBoards" | "GroupMembers" | "GroupMilestones" | "GroupProjects" | "GroupVariables" | "Epics" | "EpicIssues" | "EpicNotes" | "EpicDiscussions" | "Users" | "UserCustomAttributes" | "UserEmails" | "UserImpersonationTokens" | "UserKeys" | "UserGPGKeys" | "Branches" | "Commits" | "CommitDiscussions" | "DeployKeys" | "Deployments" | "Environments" | "Issues" | "IssueAwardEmojis" | "IssueNotes" | "IssueDiscussions" | "Jobs" | "Labels" | "MergeRequests" | "MergeRequestAwardEmojis" | "MergeRequestDiscussions" | "MergeRequestNotes" | "Pipelines" | "PipelineSchedules" | "PipelineScheduleVariables" | "Projects" | "ProjectAccessRequests" | "ProjectBadges" | "ProjectCustomAttributes" | "ProjectImportExport" | "ProjectIssueBoards" | "ProjectHooks" | "ProjectMembers" | "ProjectMilestones" | "ProjectSnippets" | "ProjectSnippetNotes" | "ProjectSnippetDiscussions" | "ProjectSnippetAwardEmojis" | "ProtectedBranches" | "ProtectedTags" | "ProjectVariables" | "Repositories" | "RepositoryFiles" | "Runners" | "Services" | "Tags" | "Triggers" | "Todos" | "PushRule" | "ApplicationSettings" | "BroadcastMessages" | "Events" | "FeatureFlags" | "GeoNodes" | "GitignoreTemplates" | "GitLabCIYMLTemplates" | "Keys" | "Licence" | "LicenceTemplates" | "Lint" | "Namespaces" | "NotificationSettings" | "Markdown" | "PagesDomains" | "Search" | "SidekiqMetrics" | "Snippets" | "SystemHooks" | "Version" | "Wikis">;
                                ~~~~~~


Found 1039 errors.
```

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/// <reference types="../typings" />
```
Comment (casz) [inline src/index.ts]: What? really? that simple? :angry:

**Comment by rkrasiuk** [comment]
@casz the linter does not highlight missing types now when I open `dist/index.d.ts`, so there should be no more problems

**Comment by casz** [comment]
It wasn't that I doubted you. Though both @jdalrymple and I spent time being frustrated :laughing: If I may say so @jdalrymple :sweat_smile:

**Comment by jdalrymple** [comment]
:upside_down_face: I wasted so many hours. Also sorry about the linting problem, I opened an issue with the semantic-release changelog repo to see how I can fix it for the future.

**Comment by jdalrymple** [comment]
So its good to be merged?

**Comment by rkrasiuk** [comment]
@jdalrymple totally

**Comment by casz** [comment]
@jdalrymple you could exclude the changelog for linting for a time being.

**Pull request merged**